launcher/FileSystem.cpp

```diff
@@ -369,10 +369,19 @@ QString getDesktopDir()
 bool createShortcut(QString destination, QString target, QStringList args, QString name, QString icon)
 {
 #if defined(Q_OS_MACOS)
+    QString invalidMacOSChars = ":";
+    for (int i = 0; i < name.length(); i++) {
```

Comment:
this loop may be better written with `std::any_of`
```cpp
   if (std::any_of(name.begin(), name.end(), [&](QChar const& c ){ return invalidOSChars.contains(c); }))
        return false;
```

Reply:
done in https://github.com/PrismLauncher/PrismLauncher/pull/826/commits/fd2a8efde8f59f3db1bdb28832ad3e5f7b2e70f0